### ISSUE TITLE
Grepros release is broken on noetic

### ISSUE THREAD

**Issue by Rayman** [suurjaak/grepros#1]
```
docker run -it ros:noetic
apt update && apt install ros-noetic-grepros
grepros -h
Traceback (most recent call last):
  File "/opt/ros/noetic/bin/grepros", line 15, in <module>
    from grepros import main
  File "/opt/ros/noetic/lib/python3/dist-packages/grepros/main.py", line 24, in <module>
    from . import plugins
ImportError: cannot import name 'plugins' from 'grepros' (/opt/ros/noetic/lib/python3/dist-packages/grepros/__init__.py)
```

**Comment by suurjaak** [comment]
Confirmed. Will soon make a new Noetic release that should fix this. 

Thank you for reporting this.

**Comment by suurjaak** [comment]
`ros-noetic-grepros` release fixed with v0.4.4.
